src/EnvChecker.php

```diff
@@ -63,4 +61,42 @@ protected function checkUndocumented(array $meta): array
             $meta
         );
     }
+
+    protected function complain(array $errors): void
+    {
+        if (0 === sizeof($errors)) {
+            return;
+        }
+
+        $this->registerExceptionHandler();
+
+        throw new InvalidEnvException(implode('; ', $errors));
+    }
+
+    protected function registerExceptionHandler()
+    {
+        if ('cli' === php_sapi_name() || ini_get('display_errors')) {
+            return;
+        }
```

Comment:
There is no need to output html to the console and no need to output to the browser when PHP is set to display_errors.